addons/mail/tests/common.py

```diff
@@ -1507,3 +1507,11 @@ def _generate_attachments_data(count, res_model, res_id, attach_values=None, pre
             'res_id': res_id,
             **attach_values,
         } for x in range(count)]
+
+    def _persona_to_store(self, data):
```

Comment:
What about having a slightly more meaningful name like `_filter_installed_module_fields`?.

`_to_store` methods make sense as they are in global scope of core models, but here you have store-formatted data as input and you filter them using a discuss-specific test tools. It's easier to understand what this does when being used with a name similar to the one I suggest.